src/components/timeline/TrackMemoryGraph.js

```diff
@@ -338,27 +348,43 @@ class TrackMemoryGraphImpl extends React.PureComponent<Props, State> {
   };
 
   _renderTooltip(counterIndex: number): React.Node {
-    if (this.props.accumulatedSamples.length === 0) {
+    const { accumulatedSamples, counter } = this.props;
+    if (accumulatedSamples.length === 0) {
       // Gecko failed to capture samples for some reason and it shouldn't happen for
       // malloc counter. Print an error and bail out early.
       throw new Error('No accumulated sample found for memory counter');
     }
-    const { minCount, countRange, accumulatedCounts } =
-      this.props.accumulatedSamples[0];
+    const { minCount, countRange, accumulatedCounts } = accumulatedSamples[0];
     const bytes = accumulatedCounts[counterIndex] - minCount;
+    const { number } = counter.sampleGroups[0].samples;
```

Comment:
I don't love the names for these properties :/ Could we make it more confusing that `number` and `count`?
Anyway, not your fault... But maybe you could name the local variable `allocationsCounters`? What do you think?